### PR TITLE
Update build docs

### PR DESCRIPTION
**Pull request by pintaf**

### CODE REVIEW

**Comment by pintaf** [comment]
I had to force push again to sign correctly.
I'll succeed to sign correctly in the first attempt. At least I hope hahaha

**Comment by utzcoz** [comment]
@pintaf, the first two commits are empty, can you squash them with the last commit?

**Comment by pintaf** [comment]
Yes the first two where just test to see if I was signing it correctly. I'm new to the squash, I'll see what I can do

**Comment by pintaf** [comment]
instead of squashing them, I deleted the useless commits. tell me if you need any other corrections

File: docs/developer/build.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ There are usually proprietary binary drivers ("blobs") you'll need to download t
...
+For example, let's say you want to build Maru OS v0.6 for Nexus 5X (bullhead). v0.6 is based on android-8.1.0_r52 (OPM7.181205.001), so you would download the [bullhead OPM7.181205.001 binaries](https://developers.google.com/android/drivers#bullheadopm7.181205.001).
 
-*Tip: To find the right binary version to select, check the manifest file that you used to sync your workspace. You can find the AOSP tag that Maru OS is based on in the manifest's [default.xml](https://github.com/maruos/manifest/blob/master/default.xml#L10). You can correlate this to the build code listed on the Nexus Binaries page [here](https://source.android.com/source/build-numbers.html).*
+*Tip: To find the right binary version to select, check the manifest file that you used to sync your workspace. You can find the AOSP tag that Maru OS is based on in the manifest's [default.xml](https://github.com/maruos/manifest/blob/maru-0.6/lineage/default.xml#L14). (Verify that the branch you're on is the same as the version of MaruOS you wanna build)You can correlate this to the build code listed on the Nexus Binaries page [here](https://source.android.com/source/build-numbers.html).*
```
Comment (pdsouza) [inline docs/developer/build.md:34]: Missing a space between parentheses and the next sentence.

**Pull request merged**